docs/DeveloperGuide.md

```diff
@@ -369,10 +372,12 @@ Command and Parser makes use of Substitutability:
 * `EatRecipeCommand` extends `Command`
 * `EatRecipeCommandParser` implements `Parser<EatRecipeCommand>` <br><br>
 
-The following activity diagram shows how eat recipe operation works:
+The following two diagrams shows how eat recipe operation works when `execute(eatR 1)` is called:
```

Comment:
this one should add the "" so its execute("eatr 1")?

Reply:
ok